SwiftIpfsApiTests/CancellableRequestTests.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  CancellableRequestTests.swift
+//  SwiftIpfsApiTests
+//
+//  Created by Marcel Voß on 17.04.20.
+//  Copyright © 2020 Teo Sartori. All rights reserved.
+//
+
+import XCTest
+
+class CancellableRequestTests: XCTestCase {
+
+    func testCancellableRequest() {
```

Comment:
This tests the mock but is missing the test for the `CancellableDataTask` struct.